### PR TITLE
Documentation fix for Python-modules: generators, GEXFIO, gephi, globals

### PR DESCRIPTION
**Pull request by fabratu**
Follow up PR of #870.

### CODE REVIEW

File: networkit/generators.pyx [hunk 1]
```diff
@@ -370,24 +513,97 @@ For a temperature of 0, the model resembles a unit-disk model in hyperbolic spac
...
+		capacity : int
+			Tuning parameter.
+		"""
 		(<_HyperbolicGenerator*>(self._this)).setLeafCapacity(capacity)
 
 	def setBalance(self, balance):
+		"""
+		setBalance(balance)
+
+		Set the balance of the quad tree. Value should be between 0.0 and 1.0.
```
Comment (angriman) [inline networkit/generators.pyx:532]:
```suggestion
		Set the balance of the quadtree. Value should be between 0.0 and 1.0.
```

File: networkit/generators.pyx [hunk 2]
```diff
@@ -370,24 +513,97 @@ For a temperature of 0, the model resembles a unit-disk model in hyperbolic spac
...
 	def setTheoreticalSplit(self, theoreticalSplit):
+		"""
+		setTheoreticalSplit(theoreticalSplit)
+
+		When using a theoretically optimal split, the quadtree will be flatter, 
+		but running time usually longer.
+
+		Parameters
+		----------
+    	theoreticalSplit : bool
```
Comment (angriman) [inline networkit/generators.pyx:551]: Indentation looks wrong here.

File: networkit/generators.pyx [hunk 3]
```diff
@@ -743,23 +1008,56 @@ cdef class LFRGenerator(Algorithm):
...
+		Parameters
+		----------
+		path : str
+			String containing a path.
+		"""
 		cls.paths["refImplDir"] = path
 
 
 	@classmethod
 	def fit(cls, Graph G, scale=1, vanilla=False, communityDetectionAlgorithm=PLM, plfit=False):
```
Comment (angriman) [inline networkit/generators.pyx:1038]: It's a bit weird to have `cls` instead of `self`. Should we change it? Maybe on a separate PR.

Comment (fabratu) [reply networkit/generators.pyx:1038]: I also wondered - maybe indeed split that into another PR.

File: networkit/gephi/pyclient.py [hunk 4]
```diff
@@ -33,85 +33,220 @@
...
+		"""
+		self.unflushedDumps = self.unflushedDumps + 1
+		if self.unflushedDumps > self.autoflush:
+			self.flush()
+			self.unflushedDumps = 0
+	def _send(self, data):
+		print('passing')
+		pass
+
+	def add_node(self, id, flush=True, **attributes):
```
Comment (angriman) [inline networkit/gephi/pyclient.py:91]: These function names could also be changed to camelCase

Comment (fabratu) [reply networkit/gephi/pyclient.py:91]: Right, should I deprecate the previous ones (since it is only a helper class)?

Comment (angriman) [reply networkit/gephi/pyclient.py:91]: Yes, but better to do that in a separate PR

**Review by angriman — APPROVED** [review]

**Pull request merged**